S3/English/ASS_33.ass

```diff
@@ -0,0 +1,108 @@
+Dialogue: 0,0:02:58.96,0:02:59.69,Default,,0,0,0,,Happy birthday
+Dialogue: 0,0:02:59.89,0:03:00.87,Default,,0,0,0,,Thank you Lily
+Dialogue: 0,0:03:00.98,0:03:02.33,Default,,0,0,0,,I'll make it my ringtone later
+Dialogue: 0,0:03:02.97,0:03:03.56,Default,,0,0,0,,I got here on time
+Dialogue: 0,0:03:04.14,0:03:05.45,Default,,0,0,0,,Happy birthday, Lynn
+Dialogue: 0,0:03:05.92,0:03:07.74,Default,,0,0,0,,This is a gift I just brought from abroad
+Dialogue: 0,0:03:07.92,0:03:08.95,Default,,0,0,0,,The newest vacuum cleaner
+Dialogue: 0,0:03:08.95,0:03:10.73,Default,,0,0,0,,We got nothing to give, what should we do
+Dialogue: 0,0:03:11.04,0:03:12.34,Default,,0,0,0,,Give anything we have
+Dialogue: 0,0:03:13.34,0:03:14.34,Default,,0,0,0,,Happy birthday Lynn
+Dialogue: 0,0:03:14.45,0:03:15.55,Default,,0,0,0,,Here's a diary of this year
+Dialogue: 0,0:03:15.79,0:03:17.26,Default,,0,0,0,,Two-thirds of the year has passed
+Dialogue: 0,0:03:17.38,0:03:17.77,Default,,0,0,0,,Yeah
+Dialogue: 0,0:03:18.23,0:03:19.77,Default,,0,0,0,,So I ripped them off for you
+Dialogue: 0,0:03:20.00,0:03:21.15,Default,,0,0,0,,Happy birthday Lynn
+Dialogue: 0,0:03:21.45,0:03:22.53,Default,,0,0,0,,A dog food bowl for you
+Dialogue: 0,0:03:22.64,0:03:23.78,Default,,0,0,0,,Isn't this your bowl
+Dialogue: 0,0:03:24.56,0:03:25.78,Default,,0,0,0,,Lynn, this hell dirt...
+Dialogue: 0,0:03:25.89,0:03:26.63,Default,,0,0,0,,Leave it for yourself
+Dialogue: 0,0:03:27.35,0:03:27.85,Default,,0,0,0,,Out of the way
+Dialogue: 0,0:03:28.25,0:03:28.85,Default,,0,0,0,,Lynn
+Dialogue: 0,0:03:29.28,0:03:30.52,Default,,0,0,0,,Here's the new clothes for you
+Dialogue: 0,0:03:31.20,0:03:32.22,Default,,0,0,0,,Happy birthday
+Dialogue: 0,0:03:32.28,0:03:33.07,Default,,0,0,0,,What's this
+Dialogue: 0,0:03:33.12,0:03:34.11,Default,,0,0,0,,Get these off
+Dialogue: 0,0:03:34.20,0:03:35.10,Default,,0,0,0,,Don't do that
+Dialogue: 0,0:03:35.10,0:03:36.29,Default,,0,0,0,,And I have a free makeup for you
+Dialogue: 0,0:03:36.29,0:03:37.00,Default,,0,0,0,,I don't need that
+Dialogue: 0,0:03:41.12,0:03:42.82,Default,,0,0,0,,I know you all forgot today is my birthday
+Dialogue: 0,0:03:45.49,0:03:47.05,Default,,0,0,0,,You don't have to bother yourself for the gift if you don't have one
+Dialogue: 0,0:03:47.72,0:03:50.35,Default,,0,0,0,,I'm already happy if you make less trouble
+Dialogue: 0,0:03:50.64,0:03:51.30,Default,,0,0,0,,Lynn
+Dialogue: 0,0:03:51.45,0:03:52.83,Default,,0,0,0,,You are so busy this year
+Dialogue: 0,0:03:52.93,0:03:54.05,Default,,0,0,0,,We won't do this in the future
+Dialogue: 0,0:03:54.05,0:03:55.00,Default,,0,0,0,,Don't say that if you can't do it
+Dialogue: 0,0:03:56.76,0:03:58.10,Default,,0,0,0,,Could we...
+Dialogue: 0,0:04:01.71,0:04:03.35,Default,,0,0,0,,Let's make an exception today
+Dialogue: 0,0:04:03.40,0:04:04.75,Default,,0,0,0,,Hooray
+Dialogue: 0,0:04:16.79,0:04:18.18,Default,,0,0,0,,It's their happiness
+Dialogue: 0,0:04:18.25,0:04:19.30,Default,,0,0,0,,I got nothing
```

Comment:
Official subs has mistimed line 107, should I follow the vocal or their sub? Currently I did the latter

Reply:
Depends on how long it is (bc of character/seconds) and if it's not too mistimed (as in moved enough for the viewer not to understand which written line correspond to the one being said)
Personally, I always fit my subs to the official ones but bc it bothers me to have them not appear and disappear exactly at the same time.

Reply:
> Depends on how long it is (bc of character/seconds) and if it's not too mistimed (as in moved enough for the viewer not to understand which written line correspond to the one being said) Personally, I always fit my subs to the official ones but bc it bothers me to have them not appear and disappear exactly at the same time.

It's mistimed for about 0.6s

Reply:
So I looked at the episode and it's not mistimed enough for it to be that ununderstandable, so if by following the vocal it ends up being on the next line, follow the sub